specs/protocol/revenue-sharing.md

```diff
@@ -0,0 +1,48 @@
+# Revenue Sharing
+
+Revenue sharing is the process by which chains in the superchain contribute a portion of their revenue or profit to the Optimism collective. They do this in return for support with the OP Stack and other benefits. 
+
+
+## Definitions
```

Comment:
latex doesn't render in mdbook but it renders in github markdown. mathjax renders in mdbook but not github markdown. Not sure which to prefer

Reply:
https://github.com/lzanini/mdbook-katex

Reply:
Ok if this works then my bad! I was having issues for some reason with latex previously